ironic.conf

```diff
@@ -5,7 +5,7 @@ default_boot_interface = ipxe
 default_deploy_interface = direct
 default_inspect_interface = inspector
 default_network_interface = noop
-enabled_boot_interfaces = pxe,ipxe,fake,redfish-virtual-media
+enabled_boot_interfaces = pxe,ipxe,fake,redfish-virtual-media,idrac-redfish-virtual-media
 enabled_deploy_interfaces = direct,fake
 enabled_hardware_types = ipmi,idrac,irmc,fake-hardware,redfish
```

Comment:
This needs `idrac-redfish`, no?

Reply:
was already present, looks like I need to rebase.